distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc

```diff
@@ -21,37 +21,38 @@
 |
 
 |metadata-content-type *[deprecated]*
```

Comment:
Maybe a note could added on the deprecated terms about what is used instead, or something like that.